client/me/account/main.jsx

```diff
@@ -217,6 +219,12 @@ class Account extends Component {
 		this.setState( { userLoginConfirm: event.target.value } );
 	};
 
+	updateSitesAsLandingPage = ( checked ) => {
```

Comment:
Can be moved there to encapsulate everything, I was looking at some previous implementation  but It might be done that way for testing purposes so there's less dependencies to mock if we write test for the component.